### PR TITLE
Dont change non path on win32

### LINKED ISSUE #106: cross-env replaces `:` for variables that aren't PATH
On Windows, if you try to use visionmedia/debug with cross-env, you will have a Bad Time, because now all of your `mynamespace:*` gets turned into `mynamespace;*` which doesn't work. cross-env should really only replace `:` => `;` for `PATH`, no other variable uses this convention

### PR DESCRIPTION
**Pull request by paulcbetts**
Closes #106

### CODE REVIEW

File: src/variable.js [hunk 1]
```diff
@@ -1,14 +1,26 @@
 import isWindows from 'is-windows'
 
+const pathLikeEnvVarWhitelist = ['PATH', 'NODE_PATH'].reduce(
```
Comment (DanReyLop) [inline src/variable.js]: This seems overly complicated. What about simply `new Set(['PATH', 'NODE_PATH'])`?

Comment (paulcbetts) [inline src/variable.js]: TIL

File: .all-contributorsrc [hunk 2]
```diff
@@ -111,6 +111,16 @@
...
       ]
+    },
+    {
+      "login": "paulcbetts",
+      "name": "Paul Betts",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/1396?v=3",
+      "profile": "https://twitter.com/paulcbetts",
+      "contributions": [
+        "bug",
+        "code"
```
Comment (kentcdodds) [inline .all-contributorsrc]: Missed `test`. You can add it in another PR if you want :)

**Comment by paulcbetts** [comment]
@kentcdodds Works for me!

**Pull request merged**